2023-12-11 12:01:31 ERROR Unable to initialize WebDriver
2023-12-11 12:01:31 ERROR 
Traceback (most recent call last):
  File "D:\a\SeleniumCapital\SeleniumCapital\pages\base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\SeleniumCapital\SeleniumCapital\pages\base_page.py", line 609, in wait_for_target_url
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\runneradmin\AppData\Local\pypoetry\Cache\virtualenvs\testcapitalcom-08IAlZho-py3.11\Lib\site-packages\selenium\webdriver\support\wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 

2023-12-11 12:01:42 ERROR Unable to initialize WebDriver
2023-12-11 12:01:42 ERROR 
Traceback (most recent call last):
  File "D:\a\SeleniumCapital\SeleniumCapital\pages\base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\SeleniumCapital\SeleniumCapital\pages\base_page.py", line 609, in wait_for_target_url
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\runneradmin\AppData\Local\pypoetry\Cache\virtualenvs\testcapitalcom-08IAlZho-py3.11\Lib\site-packages\selenium\webdriver\support\wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: